frontend/src/metabase/entities/schemas.js

```diff
@@ -82,6 +98,26 @@ export default createEntity({
   },
 });
 
+function getPreviousSchemaContainingTheQuestion(
+  state,
+  schemaId,
+  virtualQuestionId,
+) {
+  return Object.entries(state).find(([id, schema]) => {
```

Comment:
Use `Object.entries` rather than `Oject.values` + `schema.id` to not fail this test
https://github.com/metabase/metabase/blob/52346e4d2fd25181b3ef41e0624d5ec6d1302b4d/frontend/test/metabase/entities/schemas.unit.spec.js#L224